server/pools/tests.py

```diff
@@ -299,32 +303,32 @@ def test_delete(self):
         self.check_if_is_deleted(reservation_id[0])
 
         for i in range(1, total_res):
-            reservation_num = len(Reservation.objects.filter(id=reservation_id[i]))
-            self.assertEqual(reservation_num, 1)
+            reservation = Reservation.objects.get(id=reservation_id[i])
+            self.assertFalse(reservation.is_canceled)
 
     def test_double_delete(self):
         res_id = self.create_reservation()
         response = self.get_response(res_id)
         self.assertEqual(response.status_code, status.HTTP_204_NO_CONTENT)
         self.check_if_is_deleted(res_id)
         response = self.get_response(res_id)
-        self.assertEqual(response.status_code, status.HTTP_404_NOT_FOUND)
+        self.assertEqual(response.status_code, status.HTTP_204_NO_CONTENT)
+        self.check_if_is_deleted(res_id)
 
     def test_never_existed(self):
         res_id = self.create_reservation()
-        self.check_if_is_deleted(res_id + 1)
+        self.assertEqual(len(Reservation.objects.filter(id=res_id + 1)), 0)
```

Comment:
Czemu res_id+1? Co to ma sprawdzać?

Reply:
ponieważ jako id są kolejne liczby w bazie, ostatnim id jest res_id, więc res_id +1 nie istnieje,
test ten sprawdza, czy w przypadku podania złego id rezerwacji zostanie zwrócony kod 404